README.md

```diff
@@ -1,2 +1,8 @@
 # coding-review
 Project to learn coding review
+
+## Acknowledgments
+
+- Thanks to Wander Gonzalez [@wandergithub](https://github.com/wandergithub) for the project content to be reviewed.
+- Inspiration
+- etc
```

Comment:
[REQUIRED] It is great you've implemented the README and you acknowledge the author of the project content. It would be nice if you add all of the sections to match this [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md). Please open the link, check those sections and try to make your README file match that as this is required for all headers to be part of your project's README.

Reply:
👍 👍